addons/sale/models/sale_order_line.py

```diff
@@ -410,7 +410,9 @@ def _onchange_suggest_packaging(self):
             self.product_packaging_id = False
         # suggest biggest suitable packaging
         if self.product_id and self.product_uom_qty and self.product_uom:
-            self.product_packaging_id = self.product_id.packaging_ids.filtered('sales')._find_suitable_product_packaging(self.product_uom_qty, self.product_uom)
+            product_packaging_id = self.product_id.packaging_ids.filtered('sales')._find_suitable_product_packaging(self.product_uom_qty, self.product_uom)
+            if product_packaging_id:
+                self.product_packaging_id = product_packaging_id
```

Comment:
This change makes it so if you reduce the quantity to below the packaging amount, then the packaging will still remain :cry: i.e. this will break the intended behavior. Looking over the opw ticket, does it make sense to have a non-integer packaging quantity? How can you have something like .2 of of product packaging?

Reply:
We should not change it. It sounds like a better option for them (if they have the Inventory app installed) would be to use the "Package" setting because then they can indicate a "Package Type" for the packages in their delivery order, but it is indeed a lot more work/clicking for them.

Reply:
Alright, thanks